### ISSUE TITLE
How can I check dynamically for inlayHints?

### ISSUE THREAD

**Issue by mosheavni** [lvimuser/lsp-inlayhints.nvim#22]
Before calling the on-attach function, I want to be able to see if the server supports inlayHints.
I can of course define a hard-coded list, but is there a way to query the language server and tell if it provides inlay hints?
something like:

```lua
check_capability({
  bufnr = bufnr,
  client = client,
  capability = "textDocument/inlayHint"
})
``` 

Thanks.

**Comment by lvimuser** [comment]
yes, but the plugin already checks for it on https://github.com/lvimuser/lsp-inlayhints.nvim/blob/main/lua/lsp-inlayhints/core.lua#L61-L71
Special case for old versions of tsserver which used a different endpoint and didn't set the capability. Similar for jdtls, which is in the readme / commit history.
